src/renderer/screens/Operations/components/modals/ApproveTx.tsx

```diff
@@ -88,10 +90,11 @@ const ApproveTx = ({ tx, account, connection }: Props) => {
   }, [tx, signAccount?.accountId, txWeight]);
 
   useEffect(() => {
-    if (!tx.transaction || !connection.api) return;
```

Comment:
What does initial `tx.transaction` mean in this context? And why do we replace it with `getTxFromCallData(connection.api, tx.callData)`? Is that because `tx.transaction` might be invalid tx if method in calldata is unknown to spektr?

Reply:
Yes, we don't fill transaction field for unknown operation. Because it have our inner format